### ISSUE TITLE
module causes jump in page views metric.

### ISSUE THREAD

**Issue by bluxiv** [nuxt-community/google-adsense-module#104]
I recently rebuilt a website using nuxt and when it went live with this module, adsense page views metric was multiplied almost 4 times compared to the previous non-nuxt version.
revenue, impressions, clicks and cpc haven't changed in any drastic way, but i fear this change in page views could lead to problems with my account.
I have the <adsbygoogle /> component setup normally inside another component that renders it if current $route.path is not in a blacklistedUrls array.
Any help would be greatly appreciated as I've been trying to solve this problem for days to no avail.

**Comment by bluxiv** [comment]
of course, the website is live on [https://rewayat.club/](https://rewayat.club/)
I have pagelevelads and autoads enabled, with some pages having it off.

**Comment by bluxiv** [comment]
> Could you share link to your page?

**Comment by bluxiv** [comment]
> Could you share link to your page?

of course, the website is live on https://rewayat.club/
I have pagelevelads and autoads enabled, with some pages having it off.

**Comment by bluxiv** [comment]
I don't know if this is a part of the problem or not but in console, error 400 are returned more times than there're ad units.
Where i get about 5 "Failed to load resource: the server responded with a status of 400 ()" when there's only 2 ad units in the page, when i refresh the page.
and when i go to it using navigation i get about 3 errors.

**Comment by bluxiv** [comment]
I've "resolved" the problem since then, but I had to disable all nuxt routing and have all navigations through normal hrefs, which leads to a slower UX but makes the ad metrics act normally.
The problem seemed to be with ads from previous pages calling adsense on page change, let's say a page had 2 ad units it would call adsense 3 times (1 + 2) which is normal, but if I then go to another page with another 2 units it'd get called 10 times (1 + 2 + 3 + 4) this is shown in dev through console errors.
I don't know if that problem is only with my website or not but I don't think I'm doing any special implementation.
I'd love to go back to nuxt routing, but my website is doing fine atm, so take your time and thanks for everything.

**Comment by yanoyuki** [comment]
Same.

The number of page views listed in the Google Adsense report is about 2 times higher than the number of page views measured by Google Analytics.

Since I have about three Adsense ads on almost every page, the page view values should be about the same in Adsense and Analytics.

It's also very strange that the number of page views in the Google Adsense report screen is higher.

It seems like there is double counting of page views.
What could be causing this?

Any way to solve this problem without stop using `$route`, `nuxt-link`  ?

`nuxt` v2.12.2 / SSR

```package.json
...
"@nuxtjs/google-adsense": "^1.1.3",
...
```

```nuxt.config.ts

modules: [
    ...
    [
      '@nuxtjs/google-adsense',
      {
        id: ADSENSE_MEASUREMENT_ID,
        pageLevelAds: false,
      },
    ],
    ...
  ],

```

**Comment by i330z** [comment]
@yanoyuki did you found any solution for that??
